### PR TITLE
Add index on started_at and occured_at columns

### PR DESCRIPTION
**Pull request by liZe**
**To be merged after #130.**

See https://phabricator.endlessm.com/T29220.

### CODE REVIEW

File: azafea/event_processors/endless/metrics/events/_base.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ class UnknownAggregateEvent(AggregateEvent, UnknownEvent):
 class SequenceEvent(MetricEvent):
     __abstract__ = True
 
-    started_at = Column(DateTime(timezone=True), nullable=False)
+    started_at = Column(DateTime(timezone=True), nullable=False, index=True)
```
Comment (wjt) [inline azafea/event_processors/endless/metrics/events/_base.py:214]: While we're here can we add the same on `occured_at` for other event types?

Comment (liZe) [reply azafea/event_processors/endless/metrics/events/_base.py:214]: Done!

**Review by wjt — APPROVED** [review]
LGTM

**Comment by liZe** [comment]
Ready to be merged into `merge-alembic` (into #130, on top of already merged #132).

**Pull request merged**